IdentityCore/tests/util/MSIDTestCacheDataSource.h

```diff
@@ -23,7 +23,7 @@
 
 #import <Foundation/Foundation.h>
 #import "MSIDExtendedTokenCacheDataSource.h"
-#import "MSIDBaseToken.h"
+#import "MSIDCredentialType.h"
```

Comment:
This change is needed because `MSIDBaseToken.h` cannot be properly imported by the Modulemap as it creates a circular reference